src/js/components/Placeholder.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { PropTypes } from 'react';
```

Comment:
I added flow. I had to make two small tweaks to the Table components to get them to play nice:
- 0f3b5c9, made "children" optional since using `JTable` or `TableRow` in declarative fashion (within `Placeholder`) yielded errors. Googling this, it appears that the way Flow handles React children is pretty shit and there's a better workaround, unless you're aware of one?
- 2e24933, made column def's `name` optional since `processColumns` was already treating it as such.